arches/app/media/js/views/components/etl_modules/bulk-data-deletion.js

```diff
@@ -144,21 +144,25 @@ define([
             }
         });
 
+        this.deleteAlert = function() {
+            self.alert(
+                new AlertViewModel(
+                    'ep-alert-red',
+                    "Delete Data",
```

Comment:
Let's change this to blue as it's informational
```suggestion
                    'ep-alert-blue',
```